rts/Rendering/GL/glHelpers.h

```diff
@@ -0,0 +1,20 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
+#pragma once
+
+#include "myGL.h"
+#include "Rendering/Textures/TextureFormat.h"
+
+
+inline GLint glTempBindTexture(GLenum target, GLuint textureID);
+
+
+
```

Comment:
Needlessly duplicated?
```suggestion
```

Reply:
Nope, I am trying to separate declaration and definition. The list has prospects to grow.

Reply:
I may throw some separator line to make it clear.